.github/workflows/build.yml

```diff
@@ -133,11 +136,23 @@ jobs:
       run:  npm install -g cht-conf
     - name: npm CI
       run: npm ci
-    - name: horti setup
+    - name: Horti Setup
```

Comment:
Just a nitpick. 

For me, it would be easier to understand if the extra horti step, that would only execute if it's a tag, was better separated from the other one. 

I would change the "RESTART" param to a "INSTALL_LATEST" param and pass it to this extra step, and change the condition in the script to check for `IS_TAG` && `INSTALL_LATEST` to change the builds server. 

I'd also make this step be executed before `Horti Setup` and give it a suggestive name like `Install latest`.